storage/redis/client.go

```diff
@@ -117,6 +117,12 @@ func (client *Client) Delete(key storage.Key) error {
 	return nil
 }
 
+// Sync writes data to disk
+func (client *Client) Sync() error {
+	// TODO: this satisfies the storage.KeyValueStore interface, implement if needed.
+	return Error.New("Sync not implemented")
```

Comment:
So when someone uses a different backend implementation, then it will fail? The approach here should be to return `nil` because the implementation is always synced.